src/Engine/Sound/SoundManager.cpp

```diff
@@ -403,6 +407,39 @@ bool SoundManager::OnChangeBGM(const Events::ChangeBGM &e)
     return true;
 }
 
+
+bool SoundManager::OnPlayerDisconnected(const Events::PlayerDisconnected& e)
+{
+    // A player returned to the main menu. Easiest solution when we don't have states in the game. Not "PC".
+    if (m_CurrentBGMCombo != nullptr) {
+        if (getSourceState(m_CurrentBGMCombo->ALsource) == AL_PLAYING) {
+            stopSound(m_CurrentBGMCombo);
+        }
+    }
+    stopSound(m_CurrentBGMCombo);
+    Events::ChangeBGM changeBGM;
+    changeBGM.FilePath = "Audio/BGM/MenuMusic.wav";
+    m_EventBroker->Publish(changeBGM);
+    return true;
+}
+
+
```

Comment:
Listen for `Events::SetCamera` and 
`if (entity.Name() == "Overview_Camera_Start_Menu)"` set Menu music, 
`if (entity.Name() == "PickTeamCamera)"` set game background music.